src/Compilers/Core/Portable/Symbols/RefKind.cs

```diff
@@ -40,6 +40,8 @@ public enum RefKind : byte
         // NOTE: There is an additional value of this enum type - RefKindExtensions.StrictIn == RefKind.In + 1
```

Comment:
Why not update the internal value so that RefReadOnlyParameter can be set to 4?

Reply:
Simpler checking for a valid value in public APIs.

Reply:
I'd be fine to leave this as a PROTOTYPE comment, but I agree that having sequential values in the public API would be cleaner.